src/main/java/ua/lviv/iot/models/mechanical_loom_types/FlyingLoom.java

```diff
@@ -0,0 +1,24 @@
+package ua.lviv.iot.models.mechanical_loom_types;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import ua.lviv.iot.models.loom.Country;
+import ua.lviv.iot.models.loom_types.MechanicalLoom;
+import ua.lviv.iot.models.loom_types.Shape;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor
+public class FlyingLoom extends MechanicalLoom {
```

Comment:
Ok, this class has no extra information comparing to the parent one. Such classes should be removed from the codebase

Reply:
I also did it according to my UML diagram. The main purpose of creating these classes was to test how multiple inheritance works. So, now I will delete them, thanks.

Reply:
Fixed ✅